### DISCUSSION TITLE
Update to Faust Provider from sites using gqty

### DISCUSSION THREAD

**Discussion by CesarBenavides777** [wpengine/faustjs#932] [Ideas]
With the new [blog](https://faustjs.org/blog/the-future-of-faust) detailing how gqty will no longer be used, will there be an planned npx codemods to migrate current sites to new apollo framework?

**Comment by blakewilson** [comment]
Hey @CesarBenavides777 👋 

We do plan on having a migration guide from the GQty version of Faust to the Apollo version. It won't be quite as straightforward as a single `npx` command, but we will certainly have an upgrade guide!
